app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
```

Comment:
The first mention of Kong EE (or CE, for that matter) in any given document should include the full name like "Kong Enterprise Edition (EE)"